app/controllers/games_controller.rb

```diff
@@ -25,15 +25,17 @@ def show
 
   def new
     @game = Game.new
+    @game.game_records.build
```

Comment:
Why do you need to do that?

Reply:
The hacky thing is that calling `game.game_records.build` will build a single record in memory. So here you're artificially creating 6 subform for 1 game record, which is confusing code.

Instead of having `6.times` in your `erb` do that in the controller: 
```rb
6.times { @game.game_records.build }
```

Reply:
This will build 6 empty record in memory on your game. Then the `erb` template will hydrate 1 form per record.